src/Hero/Hero.spec.tsx

```diff
@@ -0,0 +1,35 @@
+// REACT
+import * as React from 'react';
+import * as renderer from 'react-test-renderer';
+// ENZYME
+import { shallow, mount } from 'enzyme';
+// COMPONENT
+import { Hero } from './Hero.component';
+const { Title, Subtitle } = Hero;
+
+// TEST SETUP
+const subject = (
+    <Hero>
+        <Title>Hero Title</Title>
+        <Subtitle>Hero Subtitle</Subtitle>
+    </Hero>
+);
+const wrapper = mount(subject);
+const component = shallow(subject);
+
+describe('Component: Hero', () => {
+    it('should be defined', () => {
```

Comment:
We should render a snapshot for (1) a hero with a header, (2) a hero with a header and a sub-header, & 3 a hero with custom content.